thirdparty/spdlog-0.13.0/cmake/Config.cmake.in

```diff
@@ -0,0 +1,24 @@
+# *************************************************************************/
+# * Copyright (c) 2015 Ruslan Baratov.                                    */

```

Comment:
Looks like we missed inclusion of Ruslan in our LICENSE for spdlog.  This was overlooked prior but we can address now.